cmd/exchange_template/exchange_template_test.go

```diff
@@ -5,12 +5,15 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/thrasher-corp/gocryptotrader/common/file"
 	"github.com/thrasher-corp/gocryptotrader/config"
 )
 
 func TestCheckExchangeName(t *testing.T) {
-	tester := []struct {
+	t.Parallel()
+	for _, value := range []struct {
```

Comment:
🐒  Change request was to change from `tester` to `tt`.
Instead we've now got `value` which is even less idiomatic.
`tt` or `tc` (for test-case) is precedented.
If we were to use `value`, idiomatic go would make it `v`.
But I'm not asking for that, I'd like `tt` or `tc`
```suggestion
	for _, tt := range []struct {
```